v3-docs/v3-migration-docs/breaking-changes/call-x-callAsync.md

```diff
@@ -32,105 +28,209 @@ Meteor.call('getAllData', (err, res) => { // [!code highlight]
 });
 
 // render data or error
-
-
 ```
 
-
 ```js [v2-server.js]
-import { Meteor } from 'meteor/meteor'
-import { Mongo } from 'meteor/mongo'
+import { Meteor } from "meteor/meteor";
+import { Mongo } from "meteor/mongo";
 
-const MyCollection = new Mongo.Collection('myCollection');
+const MyCollection = new Mongo.Collection("myCollection");
 
 Meteor.methods({
   getAllData() {
     return MyCollection.find().fetch(); // [!code highlight]
-  }
+  },
 });
 ```
 
 ```js [v3-client.jsx]
-import { Meteor } from 'meteor/meteor'
+import { Meteor } from "meteor/meteor";
 
 try {
-  const data = await Meteor.callAsync('getAllData'); // [!code highlight]
+  const data = await Meteor.callAsync("getAllData"); // [!code highlight]
   // render data
 } catch (error) {
   // render error
 }
-
 ```
 
 ```js [v3-server.js]
-import { Meteor } from 'meteor/meteor'
-import { Mongo } from 'meteor/mongo'
+import { Meteor } from "meteor/meteor";
+import { Mongo } from "meteor/mongo";
 
-const MyCollection = new Mongo.Collection('myCollection');
+const MyCollection = new Mongo.Collection("myCollection");
 
 Meteor.methods({
   async getAllData() {
     return await MyCollection.find().fetchAsync(); // [!code highlight]
-  }
+  },
 });
-
 ```
 
 :::
 
 ## Rules of using `Meteor.callAsync` & `Meteor.call`
 
-::: tip 
-It is not recommended to use concurrent calls.
-Use `await` for your `Meteor.callAsync`.
-:::
+You can still use `Meteor.callAsync` and `Meteor.call` together. But there are some catches that you need to
+pay attention to. Let's go over them.
+
+### How to NOT use them together
 
-here are a few examples of cases where you should use `Meteor.callAsync` instead of `Meteor.call`:
+These are for **async** method when called with `Meteor.callAsync` or `Meteor.call`.
+
+**Case 1**: Calling `Meteor.callAsync` and then right after calling `Meteor.call` when the methods have stubs.
 
 ```js
-import { Meteor } from 'meteor/meteor'
+import { Meteor } from "meteor/meteor";
+
+Meteor.callAsync("SOME_ASYNC_METHOD_WITH_STUB")
+  .then((data) => console.log(data))
+  .catch((err) => console.log(err));
 
-Meteor.call("someMethod", (err, res) => { // [!code error] This is not ok
+Meteor.call("SOME_ASYNC_METHOD_WITH_STUB", (err, res) => {
   if (err) {
     console.log(err);
   } else {
     console.log(res);
   }
 });
+```
 
+**Case 2**: Calling `Meteor.call` and then right after calling `Meteor.callAsync` when the methods have stubs.
```

Comment:
Is the stub used with `Meteor.call` here async? If it is, this is the same issue as with case 1 - `Meteor.call` needs to either not be used with async stubs, or should be fixed so it queues correctly with async stubs.